src/scroll-timeline-base.js

```diff
@@ -589,9 +589,9 @@ export class ViewTimeline extends ScrollTimeline {
   // explicitly set.
   constructor(options) {
     if (options.axis) {
```

Comment:
Can get rid of this special handling of axis.  ScrollTimeline and ViewTimeline both use axis instead of orientation now, and axis is already handled in the base class.